src/components/workflowEditor/workflowEditor.tsx

```diff
@@ -329,15 +332,14 @@ class WorkflowEdit extends Component<WorkflowEditProps, WorkflowEditState> {
                     <Route
                         path={[URLS.APP_LINKED_CI_CONFIG, URLS.APP_CI_CONFIG, PipelineType.WEBHOOK].map(
                             (pipeline) =>
-                                `${this.props.match.path}/${pipeline}/:ciPipelineId/cd-pipeline/:cdPipelineId?`,
+                                `${this.props.match.path}/${pipeline}/:ciPipelineId/cd-pipeline/:cdPipelineId`,
                         )}
-                        render={({ location, history, match }: { location: any; history: any; match: any }) => {
+                        render={({ location, match }: { location: any; match: any }) => {
```

Comment:
We can remove the render metjod and simplify it like child component

Reply:
We are doing some computation and sending props in render method